CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+# Contributing guidelines
+
+## Prerequisites
+- [go](https://golang.org/dl/) version v1.13+.
+- [docker](https://docs.docker.com/install/) version 17.03+.
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) version v1.14.1+.
+- [kubebuilder](https://book.kubebuilder.io/quick-start.html#installation) v2.0.1+
+
+## How to run unit tests locally
+1. Make sure kubebuilder is installed.
+2. Run `make test` in the project root directory.
+
+## Developing guidelines
+1. If CRDs in `deploy/crds` are updated, the same file names located in `deploy/olm-catalog/ibm-block-csi-operator` must be updated accordingly.
+2. If `role.yaml`, `role_binding.yaml`, or `operator.yaml` in `deploy` are updated, the ClusterServiceVersion(CSV) file in `deploy/olm-catalog/ibm-block-csi-operator` must be updated accordingly.
+3. If `README.md` is updated, ClusterServiceVersion(CSV) file in `deploy/olm-catalog/ibm-block-csi-operator` must be updated accordingly.
+4. Be sure to run `make update` or `make test` before you finish a commit.
+5. Run `operator-sdk add` to add a new api or controller, for more details, please refer to https://github.com/operator-framework/operator-sdk
+6. Run `operator-sdk generate k8s` and `operator-sdk generate openapi` after you change something in `pkg/apis`.
```

Comment:
I don't think this should be a separate step - I think that it should be a separate note at the end of the process:
NOTE: After changes are made in 'pkg/apis', be sure to run `operator-sdk generate k8s` and `operator-sdk generate openapi`

Reply:
same as above, all of them are tips, what do you think?

Reply:
I understand now what you're saying.
If that is the case, I think instead of numbered steps (1, 2, 3....) make them bullets.

Reply:
done, please have a look again.